### PR TITLE
x11-terms/zutty: Bump to 0.11

### PR DESCRIPTION
**Pull request by 00-matt**
Also clean up old versions of packages that I proxy-maintain.

### CODE REVIEW

File: x11-terms/zutty/zutty-0.11.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# It is possible to run the tests using virtualx, but it seems to take
+# screenshots of the terminal window, and compares checksums that never
+# seem to match.
+RESTRICT="test"
+
+RDEPEND="
+	media-libs/freetype:2
+	media-libs/libglvnd[X]
+	x11-libs/libXmu
+"
```
Comment (juippis) [inline x11-terms/zutty/zutty-0.11.ebuild:28]:
```
 * VDB: detected possibly incorrect RDEPEND (x11-terms/zutty-0.11)
 *  > x11-libs/libX11
```
(you should add that as a dep)
this also needs to depend on `virtual/opengl`.

File: x11-terms/zutty/zutty-0.11.ebuild [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+BDEPEND="
+	${PYTHON_DEPS}
+	virtual/pkgconfig
+"
+
+DOCS=( doc/KEYS.org doc/USAGE.org )
+
+src_prepare() {
+	# Remove default CXX/LDFLAGS, bug #830405.
+	sed -i '72,76d' wscript || die
```
Comment (juippis) [inline x11-terms/zutty/zutty-0.11.ebuild:39]: Yeah I really hate this, you should just patch it.

File: x11-terms/zutty/zutty-0.11.ebuild [hunk 3]
```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7,8,9} )
+PYTHON_COMPAT=( python3_{8,9} )
```
Comment (juippis) [inline x11-terms/zutty/zutty-0.11.ebuild:6]: 10? :)

Comment (00-matt) [reply x11-terms/zutty/zutty-0.11.ebuild:6]: Just tested with py3.10 and seems to work. Looks like a few other waf ebuilds have moved too.

```
dev-libs/serd/serd-0.30.10.ebuild:PYTHON_COMPAT=( python3_{8..10} )
dev-libs/sord/sord-0.16.8-r1.ebuild:PYTHON_COMPAT=( python3_{8..10} )
dev-libs/sord/sord-9999.ebuild:PYTHON_COMPAT=( python3_{8..10} )
media-libs/lilv/lilv-0.24.12.ebuild:PYTHON_COMPAT=( python3_{8..10} )
media-libs/sratom/sratom-0.6.8.ebuild:PYTHON_COMPAT=( python3_{8..10} )
media-libs/suil/suil-0.10.10.ebuild:PYTHON_COMPAT=( python3_{8..10} )
media-sound/ardour/ardour-6.9.ebuild:PYTHON_COMPAT=( python3_{7,8,9,10} )
media-sound/ardour/ardour-9999.ebuild:PYTHON_COMPAT=( python3_{7,8,9,10} )
media-sound/guitarix/guitarix-0.43.1.ebuild:PYTHON_COMPAT=( python3_{7..10} )
media-sound/guitarix/guitarix-9999.ebuild:PYTHON_COMPAT=( python3_{7..10} )
media-sound/jack2/jack2-1.9.19.ebuild:PYTHON_COMPAT=( python3_{8,9,10} )
media-video/mpv/mpv-0.34.0.ebuild:PYTHON_COMPAT=( python3_{8..10} )
media-video/mpv/mpv-9999.ebuild:PYTHON_COMPAT=( python3_{8..10} )
net-fs/samba/samba-4.15.2-r2.ebuild:PYTHON_COMPAT=( python3_{8..10} )
net-fs/samba/samba-4.15.3-r1.ebuild:PYTHON_COMPAT=( python3_{8..10} )
sys-libs/ldb/ldb-2.4.1.ebuild:PYTHON_COMPAT=( python3_{8..10} )
```

**Review by juippis — APPROVED** [review]
Thanks! You threw the patch outside "${FILESDIR}" but I'll update that. Works now!

**Pull request closed without merging**